examples/get-insights-data/main.py

```diff
@@ -0,0 +1,121 @@
+import os
+import sys
+import csv
+from utils.utils import PublicAPIClient, get_commandline_arguments
+
+
+def main():
+    args = get_commandline_arguments(
+        [
+            (
+                ["course_id"],
+                {"type": int, "help": "id of the course to get insights for"},
```

Comment:
it didn't like my uuid courses, but  we can get back to that once it's a real issue

Reply:
UUID for courses? I think we agreed that we use the numerical ID we have in the Canvas URL.

Reply:
We discussed uuid-s in the context of QLD/consortium preparation. But since that whole topic is shelved, I think we should stick to what we have. UUIDs for courses are not stored in studio, so it would be a larger piece of work to get them, and make things work with them.